src/data/items/resourceChemistry.js

```diff
@@ -104,6 +104,14 @@ const POTIONS = {
 		potionJob: "cooking",
 		potionCharges: 10
 	},
+	potionBartending: {
+		name: "Powerful Shaker",

```

Comment:
New name for this would be nice, probably. It doesn't really fit in with the others but I couldn't think of one.